libsolidity/analysis/ReferencesResolver.h

```diff
@@ -88,6 +88,10 @@ class ReferencesResolver: private ASTConstVisitor, private yul::ASTWalker
 	void operator()(yul::Identifier const& _identifier) override;
 	void operator()(yul::VariableDeclaration const& _varDecl) override;
 
+	/// Checks if the name contains a '.'.
+	void checkYulIdentifierName(yul::YulString _name, langutil::SourceLocation const& _location);
```

Comment:
Hm, how about `validateYulIdentifierName`?

Reply:
I'd favor validateXX or verifyXX as I feel those two words are more restrictive in their meanings than "check". But I think this may as well be personally subjective?